services/filesstore/filesstore.go

```diff
@@ -34,15 +34,16 @@ func NewFileBackend(settings *model.FileSettings, enableComplianceFeatures bool)
 	switch *settings.DriverName {
 	case model.IMAGE_DRIVER_S3:
 		return &S3FileBackend{
-			endpoint:  *settings.AmazonS3Endpoint,
-			accessKey: *settings.AmazonS3AccessKeyId,
-			secretKey: *settings.AmazonS3SecretAccessKey,
-			secure:    settings.AmazonS3SSL == nil || *settings.AmazonS3SSL,
-			signV2:    settings.AmazonS3SignV2 != nil && *settings.AmazonS3SignV2,
-			region:    *settings.AmazonS3Region,
-			bucket:    *settings.AmazonS3Bucket,
-			encrypt:   settings.AmazonS3SSE != nil && *settings.AmazonS3SSE && enableComplianceFeatures,
-			trace:     settings.AmazonS3Trace != nil && *settings.AmazonS3Trace,
+			endpoint:   *settings.AmazonS3Endpoint,
+			accessKey:  *settings.AmazonS3AccessKeyId,
+			secretKey:  *settings.AmazonS3SecretAccessKey,
+			secure:     settings.AmazonS3SSL == nil || *settings.AmazonS3SSL,
+			signV2:     settings.AmazonS3SignV2 != nil && *settings.AmazonS3SignV2,
+			region:     *settings.AmazonS3Region,
+			bucket:     *settings.AmazonS3Bucket,
+			pathPrefix: *settings.AmazonS3PathPrefix,
```

Comment:
Tests are failing and looks like is related to the fact that you are using a wrong config in the tests. Can you verify the tests are passing the new field to make it work?

Reply:
done